### PR TITLE
First request

### PR DESCRIPTION
**Pull request by Hernosepo**
Hope changes are useful!

### CODE REVIEW

File: app/utilities/im_processing.py [hunk 1]
```diff
@@ -27,4 +28,7 @@ def load_img(path, size: Union[tuple[int] | float] = None):
         new_size = tuple(int(length * size) for length in img.size)
         return ImageTk.PhotoImage(img.resize(new_size))
 
-    raise ValueError("Expected size to be either a tuble of ints, a float or None.")
+    
+#Grammar correction tuble to tuple
```
Comment (ArielMAJ) [inline app/utilities/im_processing.py:32]: You may add comments explaining what the code does, but you don't need to add comments directly in the code saying what you changed (these comments will persist and might be confusing later after this PR is closed). You can explain your changes in the Pull Request.
Please remove this one.

File: app/utilities/im_processing.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 from PIL import Image, ImageTk  # type: ignore[import]
 
 
+# Vs code is showing me this error: Union requires two or more type arguments.
```
Comment (ArielMAJ) [inline app/utilities/im_processing.py:9]: Good to know! I'm using a simpler IDE and it didn't say anything here.
Could you remove this comment and do the necessary changes? I found some information [here](https://stackoverflow.com/a/52471875)

**Review by ArielMAJ — CHANGES_REQUESTED** [review]
Please, do a "git reset --soft HEAD~2" to undo the two commits you did. Do the corrections I asked in my review, then git add and commit them again (so your PR will have only one commit). Then use "git push -f".

File: app/utilities/im_processing.py (hunk 1 above)
Comment (Hernosepo) [reply app/utilities/im_processing.py:32]: Thanks, noted and working on it.

File: app/utilities/im_processing.py (hunk 2 above)
Comment (Hernosepo) [reply app/utilities/im_processing.py:9]: I ve changed it, lets see.

**Review by ArielMAJ — COMMENTED** [review]
It seems your commits were squished into one, but the comments weren't removed nor the Union was corrected. Maybe you forgot to stage the new changes (git add)?

**Review by ArielMAJ — APPROVED** [review]

**Comment by ArielMAJ** [comment]
Thank you for working on issue #5!
Next PR make sure to mention the issue so your PR is automatically linked to it.

**Pull request merged**